applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -280,7 +280,7 @@ def __init__(self, model, custom_settings):
 
         ## Construct the turbulence model solver
         if not self.settings["turbulence_model_solver_settings"].IsEquivalentTo(KratosMultiphysics.Parameters("{}")):
-            self._turbulence_model_solver = CreateTurbulenceModel(model, self.settings["turbulence_model_solver_settings"])
+            self._turbulence_model_solver = CreateTurbulenceModel(model, self.main_model_part, self.settings["turbulence_model_solver_settings"])
```

Comment:
Why do you need both the model and the model part? Shouldn't one of them be enough?

Reply:
same question